sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -1015,6 +1031,7 @@ def store_callback(table, **kwargs):
               # Set a batch size such that the input elements will be inserted
               # in 2 batches.
               batch_size=2,
+              triggering_frequency=None,
```

Comment:
can we make sure that users are not forced to define a triggering_frequency for their pipelines unless they're using file loads in streaming?